editor/styling/stylesheets/modules/_tool-tabs.scss

```diff
@@ -42,10 +42,6 @@
         }
     }
 
-    GridPane {
-        -fx-padding: 10;
-    }
-
```

Comment:
I'm not sure why it was here, but this was messing with GridPanes used in the override inspector view. The only other visible difference I found after this change is in the "CSS Test" tool tab which is only used in dev.